### ISSUE TITLE
only marks authors intermittantly

### ISSUE THREAD

**Issue by AdamRGrey** [listen2/Imgur-Enhancement-Suite#3]
http://i.imgur.com/QbWcVyx.png No idea how you'd fix this, I'm starting to kind of hate React.js. This is also a test to see if this is still being worked on.

**Comment by listen2** [comment]
Yeah, this project is mostly abandoned. I don't have the energy to keep up with imgur's frequent frontend changes.
